packages/desktop-ui-lib/src/lib/settings/plugins/services/factories/source.factory.ts

```
@@ -0,0 +1,20 @@
+		switch (type) {
+			case PluginSourceType.CDN:
+				return new CdnSourceStrategy();
+			case PluginSourceType.NPM:
+				return new NpmSourceStrategy();
+			case PluginSourceType.GAUZY:
+				return new GauzySourceStrategy();
+			default:
+				throw new Error(`Unsupported source type: ${type}`);
+		}
+	}
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)